test/lsp/ProtocolTest.cc

```diff
@@ -98,7 +101,7 @@ vector<CounterImpl::Timing> CounterStateDatabase::getTimings(ConstExprStr counte
             }
 
             if (tagsMatched == tags.size()) {
-                rv.push_back(timing);
+                rv.emplace_back(&timing);
```

Comment:
This takes the address of an element of a vector, which might be re-allocated and invalidate the pointer.

I couldn't think of a good way around this, because the `getTimings()` helper is designed to return a subset of the timings which match some set of tags.

After the optimization to convert Timers to have unique_ptr's in them, we can't copy them around without a bunch of work, and this place was previously copying.